recipe/patch_yaml/chromadb.yaml

```diff
@@ -0,0 +1,9 @@
+# chromadb depends on uvicorn-standard not uvicorn
+# Fixed in https://github.com/conda-forge/chromadb-feedstock/pull/51
+if:
+  name: chromadb
+  timestamp_lt: 1727439944000
+then:
+  - replace_depends:
+      old: uvicorn
+      new: uvicorn-standard
```

Comment:
These are exact matches, we need some kind of pattern matching here.

Reply:
Ah. Should be "rename_<>" not "replace_<>".